rfc-090-disable-floc-tracking-by-default.md

```diff
@@ -0,0 +1,63 @@
+# Disable FLoC tracking by default
+
+## Summary
+
+Google have introduced a new (at time of writing) standard, which
+organises users into cohorts according to their browsing history
+in the browser and shares this information to advertisers. This has
+severe security risks and we should take steps to make sure any
```

Comment:
Is it a _security_ risk rather than a privacy risk? I don't see a security angle here. As far as I can tell the WordPress proposal to treat it like a security risk is more for the purposes of getting it quickly addressed.